July tech news update (#1073)
pathoplexus/pathoplexus: monorepo/website/src/pages/news/2026-07-22-july-update.mdx

 
### Sequence and SeqSet citations

Pathoplexus now integrates data from [Crossref](https://www.crossref.org/) to show how [SeqSets](https://pathoplexus.org/docs/concepts/seqset) have been used in scientific publications. You can find this information directly on the SeqSet page. See here for an example of a SeqSet of West-Nile Virus sequences: [https://pathoplexus.org/seqsets/PP_SS_200.1](https://pathoplexus.org/seqsets/PP_SS_200.1).

![An image showing the SeqSet PP_SS_200.1 on Pathoplexus as a demonstration of a SeqSet that's been cited](/gifs/news/2026-07-22-july-update/seqset_with_citation.png)

Furthermore, this information is also displayed on the sequence details page when a sequence is part of a SeqSet that has been referenced by a scientific publication:

![An image showing how citations are displayed on the sequence details page on Pathoplexus](/gifs/news/2026-07-22-july-update/sequence_with_citation.png)

> When using a SeqSet in your publications, please remember to [cite it]( https://pathoplexus.org/about/citation) in the references section so those generating and sharing data are properly credited for their work.

Reviewer: ```suggestion
When using a SeqSet in your publications, please remember to [generate a DOI](https://pathoplexus.org/docs/how-to/generate-seqset#adding-a-doi) and [cite it]( https://pathoplexus.org/about/citation) in the references section so those generating and sharing data are properly credited for their work.
```
Clarifying that citing the SeqSet isn't enough, need to generate a DOI!

Author: But just for my understanding, someone could use a pre-existing SeqSet in their work that already has a DOI, right? Is this worth making explicit in the text, or is it much more likely that everyone just creates their own SeqSets when they're doing a study (and thus will need to create a DOI)?

Reviewer: I think it's actually pretty unlikely that people reuse SeqSets - they're such specific sets of sequences to a study (which may vary over time, locations included, countries, etc, etc). It's not impossible but I think it's most likely people generate their own - and we seem to continue to have issues with people not citing things properly 🙃

### Submission workflow

#### Improved submission template
 
When submitting your data to Pathoplexus, you can now download a rich XLSX submission template that contains definitions of our metadata fields and - where appropriate - lists accepted values:

![An image showing the new XLSX submission template for Pathoplexus](/gifs/news/2026-07-22-july-update/submission_template_xlsx.jpg)

> These templates are available on the submission page or on in the docs (example ANDV): [https://pathoplexus.org/docs/concepts/metadataformat?organism=andv&fieldType=inputFields](https://pathoplexus.org/docs/concepts/metadataformat?organism=andv&fieldType=inputFields).

Reviewer: ```suggestion
These templates are available on the submission page or in the docs (example for ANDV): [https://pathoplexus.org/docs/concepts/metadataformat?organism=andv&fieldType=inputFields](https://pathoplexus.org/docs/concepts/metadataformat?organism=andv&fieldType=inputFields).
```

> These templates are available on the submission page or on in the docs (example ANDV): [https://pathoplexus.org/docs/concepts/metadataformat?organism=andv&fieldType=inputFields](https://pathoplexus.org/docs/concepts/metadataformat?organism=andv&fieldType=inputFields).

Reviewer: ```suggestion
These templates are available on the submission page or in the docs (example ANDV): [https://pathoplexus.org/docs/concepts/metadataformat?organism=andv&fieldType=inputFields](https://pathoplexus.org/docs/concepts/metadataformat?organism=andv&fieldType=inputFields).
```

#### Improvements to revision and revocation

> It is now also possible to download the original data associated with multiple sequences to simplify batch revisions. Clicking the ‘Download 15 selected entries’ button in the image below would return the original submitted data for all selected sequences:

Reviewer: ```suggestion
For sequences submitted by your group(s), it is now also possible to download the original data associated with multiple sequences to simplify batch revisions. Log in and navigate to 'View' under 'Submit Sequences' for any organism to see previously submitted sequences. Clicking the ‘Download 15 selected entries’ button in the image below would return the original submitted data for all selected sequences:
```

![An image showing multiple sequences being selected at once in the Pathoplexus interface, highlighting that original data can now be downloaded for all selected sequences](/gifs/news/2026-07-22-july-update/download_selected_entries.png)

> As before, you can also make revisions in the browser on a single sequence (no need to download anything) - but please do remember to use or download the original metadata if you are making bulk revisions!

Reviewer 2: ```suggestion
As before, you can also make revisions in the browser for a single sequence (no need to download anything) - but please do remember to use or download the original metadata if you are making bulk revisions!
```

#### Changes to our submission format
 
We now support date ranges in the Lucene format (e.g., `[2014 TO 2015]`), as well as the ISO format (e.g., `2026-01-01/2026-02-10`). If you are unable to give an exact date, this is a great way to supply useful information nonetheless!

We now validate submitted host details using NCBI taxonomy data. You can specify the host organism by providing a scientific name _or_ NCBI taxon id in the new `host` metadata field. The submitted data is validated and used to generate the `hostTaxonId`, `hostNameScientific`, and `hostNameCommon` metadata fields. This ensures consistent host designation, and powers our new hierarchical taxonomy search, covered below!

### Search improvements

#### Hierarchical taxonomy search
 
Pathoplexus now validates host metadata, enabling hierarchical taxonomy searches on host information. When a higher-level taxon is selected, the search returns all sequences whose host belongs to any descendant taxon of the selected taxon.

> This link, for example, shows all Dengue sequences observed in mosquitos, including seven species in the _Aedes_ genus and two in the _Culex_ genus: [https://pathoplexus.org/dengue/search?hostTaxonId=7157*&](https://pathoplexus.org/dengue/search?hostTaxonId=7157*&)

Reviewer: ```suggestion
This link, for example, shows all Dengue sequences observed in mosquitoes, including seven species in the _Aedes_ genus and two in the _Culex_ genus: [https://pathoplexus.org/dengue/search?hostTaxonId=7157*&](https://pathoplexus.org/dengue/search?hostTaxonId=7157*&)
```

Author: Agree with the edit in principle, but the annoyong thing is that NCBI uses 'mosquitos' as common name for Culicidae, so if someone were to type in 'mosquitoes' in the `Host species` search field it won't return anything.

Minor nit, though, and if we feel 'mosquitoes' reads nicer in the news update then we can just go with that :)

Reviewer: Ah ok fair enough, let's stick with mosquitos!

![An image showing the new hierarchical filtering interface for host organisms in Pathoplexus](/gifs/news/2026-07-22-july-update/host_species_autocomp.jpg)

#### New search features

To improve usability, search filters are now organized into collapsible sections for metadata and sequence-based filters. For multi-segmented organisms, sequence filters are further grouped by segment. A new ‘Required segments’ filter also allows users to restrict results to records that include specific genome segments.

Pathoplexus now supports search across multiple identifiers, e.g. accession, INSDC accession and BioProject accession using a single filter:

> ![An image showing Pathoplexus' new multi-field search box that users can use to search across diffferent types of identifiers](/gifs/news/2026-07-22-july-update/multi_field_search.png)

Reviewer: ```suggestion
![An image showing Pathoplexus' new multi-field search box that users can use to search across different types of identifiers](/gifs/news/2026-07-22-july-update/multi_field_search.png)
```

#### Pathoplexus Docs Search

The Pathoplexus [About](https://pathoplexus.org/about) and [Docs](https://pathoplexus.org/docs) pages now support free-text search for easier access to all types of information about Pathoplexus. This is available through the search box in the top left of these pages, which searches all pages in both sections.

> ### Convenient version comparison

Reviewer 3: I just read through and it looks really good! My only suggestion was to rename this title to just 'Version Comparison'

![An image showing where users can find the new 'Compare versions' button in the Pathoplexus interface](/gifs/news/2026-07-22-july-update/compare_versions.png)

This will take you to a page highlighting which fields have changed between versions in a side-by-side table view. You can specify which two versions of a sequence you would like to compare, as well as whether to hide fields that haven’t changed between versions or not. After configuring the comparison view to your liking, you can share it by simply copying the URL. See here for an example: [https://pathoplexus.org/seq/PP_006W6RC.3/versions?compare=2%2C3](https://pathoplexus.org/seq/PP_006W6RC.3/versions?compare=2%2C3)

![An image showing an example of the new 'Compare versions' being used to compare version 2 & 3 of a sequence](/gifs/news/2026-07-22-july-update/compare_versions_table.png)

### Further updates and improvements

Pathoplexus has added support for visualizing amino acid substitutions relative to the Ebola Bundibugyo outbreak clade founder, to allow at-a-glance information about new sequences from the ongoing outbreak.
 

> Pathoplexus now provides [linkOuts](https://pathoplexus.org/docs/how-to/use-tools-and-add-new-ones) to view sequence alignments of Pathplexus sequences using [Sealion](https://github.com/artic-network/sealion/tree/main), or to place sequences on phylogenetic trees constructed using [viral UShER](https://github.com/AngieHinrichs/viral_usher) and view them using [Taxonium](https://www.taxonium.org/).

Reviewer: ```suggestion
Pathoplexus now provides [Link Outs](https://pathoplexus.org/docs/how-to/use-tools-and-add-new-ones) to view sequence alignments of Pathoplexus sequences using [Sealion](https://github.com/artic-network/sealion/tree/main), or to place sequences on phylogenetic trees constructed using [viral UShER](https://github.com/AngieHinrichs/viral_usher) and view them using [Taxonium](https://www.taxonium.org/).
```
Use more user friendly naming, rather than the variable name :)

> Pathoplexus now provides [linkOuts](https://pathoplexus.org/docs/how-to/use-tools-and-add-new-ones) to view sequence alignments of Pathplexus sequences using [Sealion](https://github.com/artic-network/sealion/tree/main), or to place sequences on phylogenetic trees constructed using [viral UShER](https://github.com/AngieHinrichs/viral_usher) and view them using [Taxonium](https://www.taxonium.org/).

Reviewer: ```suggestion
Pathoplexus now provides [linkOuts](https://pathoplexus.org/docs/how-to/use-tools-and-add-new-ones) to view sequence alignments of Pathoplexus sequences using [Sealion](https://github.com/artic-network/sealion/tree/main), or to place sequences on phylogenetic trees constructed using [viral UShER](https://github.com/AngieHinrichs/viral_usher) and view them using [Taxonium](https://www.taxonium.org/).
```

Author: well that's embarrassing
